portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //why doesnt the headers in html format work and only show up as chars?
```

Comment:
Nit: Remove this comment. I suppose it'll show up formatted if we add some additional tags to it. Like `<h1>Hello Mariah! How are you today?</h1>`